### ISSUE TITLE
Collaboration with WoT CG

### ISSUE THREAD

**Issue by egekorkan** [w3c/wot-marketing#288]
Somewhat of a continuation of #219 

We need to decide what tasks done by the Marketing Group are better suited for CG. Please provide opinions below. Current tasks that are in the scope of the marketing TF can be found at https://www.w3.org/WoT/activities/task-forces/tf-marketing

**Comment by relu91** [comment]
> The marketing TF should be the group where official outgoing communication of the WG/IG is created and published.
> 
> Web presence of WG/IG, press releases and other official communication should remain in the marketing TF and follow the current practice of getting approved in the main call before it is sent out.
> 
> I can see the primary tasks for the CG to socialize information about implementations and drive adoption by Twitter posts, informal news, and other social media communication. Organisation of an OpenDay should also be in scope of the CG.

I tend to agree with your comment, however, I want also to minimize splitting the user base into different social accounts/web pages and communication channels. I would prefer to use the CG as "a social media manager" that takes announcements from the WG/IG by means of the Marketing TF and publish them on their pages. Of course, such updates need to be clearly marked as coming from WG/IG and not from the CG itself. Having this would still allow having a strict check for press releases from the WG but other updates can be done quickly without waiting for particular consensus (i.e. publishing examples, advertising WoT implementations, papers, etc. ).

**Comment by relu91** [comment]
So taking into account [this list](https://w3c.github.io/wot-marketing/activities/task-forces/tf-marketing/), I think the CG should be in charge to handle:
- Reaching out to the wide community through:
  * Social Media
  * Web Pages
  * Blog Posts
  * Promotional Text and Videos
  * Explainer Text and Videos
  * Tutorial Text and Videos

**Comment by egekorkan** [comment]
Call of 29.03.22:
We have identified 3 pillars of discussion to continue on:
1. Expectations: What does CG expect from the collaboration with the IG and what does IG expect from CG?
2. Tools: What tools does CG want to use that are normally given to IGs and WGs? These include repositories, webex, irc
3. Working style: How does the CG work and how does CG keep in sync with IG?

Since these are all discussions on their own, we have decided to close this issue and separate the discussion there. These issues will have include `[CG Collab]` in the issues title and labeled `CG`.
